src/pages/Filing/FilingApp/FileSubmission.tsx

```diff
@@ -151,33 +193,45 @@ export function FileSubmission(): JSX.Element {
                 <Input
                   type='file'
                   ref={fileInputReference}
+                  title={buttonLabel}
                   className='absolute inset-0 h-full w-full cursor-pointer opacity-0'
                   id='file-input-specific'
                   name='file-input-specific'
                   accept='.csv'
-                  aria-describedby='file-input-specific-hint'
+                  aria-hidden='true' // Hidden from screenreaders
```

Comment:
Ok, so my big comment here is that the input should be the thing that screen readers interact with, not the button you have below. At other gigs, I've styled the file input to look like a button, which works ok. Take a peek at the [USWDS](https://designsystem.digital.gov/components/file-input/) and its implementations for inspiration here.